core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/SparkRunner.scala

```diff
@@ -212,7 +217,7 @@ object SparkResource {
   )
 
   final case class Driver(
-      cores: Option[Float] = None,
+      cores: Option[Int] = Some(1),
```

Comment:
Are you sure this is not a float value, like `cpu` in K8s resource requests/limits?

Reply:
Yes it has to be an Int, as indicated in the URL in my comment above. Using a float will make our Spark app (e.g. spark-sensors) fail with a CR validation error.